lib/tests/test_mut_repo.rs

```diff
@@ -715,3 +716,52 @@ fn test_reparent_descendants() {
         }
     }
 }
+
+#[test]
+fn test_bookmark_hidden_commit() {
+    // Test that MutableRepo::set_local_bookmark_target() on a hidden commit updates
+    // the view head ids as setting it makes the commit visible.
```

Comment:
"updates the view head ids as setting it makes the commit visible" sounds backwards (it becomes visible because we make it a visible head). I think just "makes it visible" is enough

Reply:
done